pkg/connector/document.go

```diff
@@ -58,13 +60,29 @@ func (o *documentBuilder) List(ctx context.Context, parentResourceID *v2.Resourc
 				return nil, "", nil, err
 			}
 		}
+		l.Info("baton-lucidchart: folder content", zap.Any("content", folderContent))
 
-		innerDocuments, err := documentResources(folderContent, parentResourceID)
-		if err != nil {
-			return nil, "", nil, err
+		var resources []*v2.Resource
+		for _, item := range folderContent {
+			if item.Type != "document" {
+				continue
+			}
+
+			if item.IsShortcut && !o.includeShortcuts {
```

Comment:
This is a breaking change. The option should be the other way around. If you don't want short cuts they should be excluded. You are removing previously synced data from customers. Yes, Zscaler will have to enable the option to unbreak their sync, but in this scenario it is not a breaking change that affects other customers. We would would have to investigate all of the customers using lucidcharts to understand whether or not they use shortcuts to validate if this will cause a problem otherwise.